### ISSUE TITLE
Evaluate SQLite libraries for OpenCodelists

### ISSUE THREAD

**Issue by iaindillingham** [opensafely-core/opencodelists#1282]
tl;dr No action required. We should stick with `pysqlite3-binary`/https://github.com/coleifer/pysqlite3.

The issue, from [this Slack thread](https://bennettoxford.slack.com/archives/C03FB777L1M/p1655284433640079):

>I’ve been setting up OpenCodelists on macOS. Why do we use `pysqlite3-binary` in preference to `pysqlite-binary`? As well as more information on PyPI, [the latter](https://pypi.org/project/pysqlite-binary/) has wheels for Windows and macOS, as well as Linux. The projects seem more-or-less identical — both include The Institute’s SQLite Advocate-In-Chief [@dave](https://bennettoxford.slack.com/team/UAXE5V4RG) as a member. (I see from the Justfile that I could setup OC for Docker development, which is fine. But if all that stands in the way is a switching one dependency for another, then I’ll open a PR.)

* [`pysqlite3-binary`](https://pypi.org/project/pysqlite3-binary/). Used by OpenCodelists, no wheels for Windows and macOS. https://github.com/coleifer/pysqlite3.

* [`pysqlite-binary`](https://pypi.org/project/pysqlite-binary/). Wheels for Windows, macOS, and Linux. https://github.com/pysqlite3/pysqlite3

Notice that in both cases, the repo is pysqlite3. However, the org is different. We'll disambiguate using the org: that is, using `coleifer` and `pysqlite3`.

When was the fork?

```
git clone git@github.com:coleifer/pysqlite3.git coleifer
cd coleifer/
git remote add --fetch pysqlite3 git@github.com:pysqlite3/pysqlite3.git
git log --oneline master..pysqlite3/main
```

`pysqlite3` is 17 commits ahead of `coleifer`. The former was forked from the latter after v0.4.7 was tagged (`8900b3d`). The new commits add a GHA workflow for building Wheels for Windows, macOS, and Linux.

coleifer/pysqlite3#13 discusses GHA, but was closed. coleifer/pysqlite3#7 discusses wheels for macOS, but was closed.

[Tim Zhang](https://github.com/ttimasdf) opened coleifer/pysqlite3#37 before forking. The response from Charles Leifer was... blunt...

>Sorry, definitely not interested.

So, I conclude that the fork is a 🖕.

Whilst it would be nice to have a local (i.e. non-Docker) development environment for Windows and macOS, OpenCodelists does have a really good Docker development environment. So, I don't think we need the fork.

**Comment by evansd** [comment]
Thanks @iaindillingham, that's really helpful and I agree with your conclusion. Hopefully the dust will eventually settle around this and, one way or the other, we'll end up with binary wheels for Windows and macOS. But I agree that we shouldn't make any changes now.
